nemo/collections/asr/models/ctc_models.py

```diff
@@ -596,6 +597,22 @@ def training_step(self, batch, batch_nb):
 
         return {'loss': loss_value, 'log': tensorboard_logs}
 
+    def predict_step(self, batch, batch_idx, dataloader_idx=0):
+        signal, signal_len, transcript, transcript_len, sample_id = batch
+        if isinstance(batch, DALIOutputs) and batch.has_processed_signal:
```

Comment:
I see we dont support dali for returning index. Could you take a look to add it? Or note that we wont get DALIOutputs here.

Reply:
Ok sounds good, lets leave Dali for later.